tests/_helpers.py

```diff
@@ -0,0 +1,17 @@
+from collections.abc import Generator
```

Comment:
❓ Not important at all & just out of curiosity: Do we need the underscore in the module name? Tried Googling it and didn't come across anything.

Reply:
We don't, can rename